src/ClimateMachine.jl

```diff
@@ -33,12 +33,14 @@ include(joinpath(
 include(joinpath("Numerics", "SystemSolvers", "SystemSolvers.jl"))
 include(joinpath("Numerics", "ODESolvers", "ODESolvers.jl"))
 include(joinpath("Numerics", "ODESolvers", "GenericCallbacks.jl"))
-include(joinpath("Atmos", "Model", "AtmosModel.jl"))
 include(joinpath("InputOutput", "VTK", "VTK.jl"))
+include(joinpath("Atmos", "Model", "AtmosModel.jl"))
+include(joinpath("Common", "TurbulenceClosures", "TurbulenceClosures.jl"))
```

Comment:
I've seen problems arising from ordering in this list. As far as possible, it's best to include a module A that is used by a module B before including module B. If it's working as is, then it doesn't matter.